ioos_qc/config.py

```diff
@@ -182,7 +182,7 @@ def run(self, **passedkwargs):
 def extract_calls(source) -> List[Call]:
     """Extracts call objects from a source object.
 
-    Args:
+    Args
```

Comment:
My guess is that the warning was due to a shorter underline of `_`, right? If so, why remove the `:` instead of adding another `_` below?

Reply:
Adding shorter underline do removed the warning. But still it leads to unexpected indentation error.

Reply:
You mean longer?

Reply:
Earlier there are 2 warnings and 1 error:
Warning1: ```Block quote ends without a blank line; unexpected unindent.```
Warning2: ```TITILE: Underline too short```

Error: ```Unexpected indentation```

If I do this: 
```
Args:
-----
```

Then it can only resolve warning2.

Reply:
For 1 you probably need to dedent the block of text below, no?

Reply:
We can do that, but still the issue persists.

Reply:
@Sakshamgupta90 I'm not sure I follow. I get no warnings with something like:

```
    """Aggregates an array of flags by precedence into a single array.

    Args:
    -----
        vectors: An array of uniform length arrays representing individual flags

    Returns:
    --------
        A masked array of aggregated flag data.

    """
```

Keeping both the `:` and the underlines and having a correct HTML rendering.

Reply:
Yes, the indentation from the first two args is differente from the last 3. those require fixing as well.

Reply:
Yes, I tried to fix that but no positive result shown. So I tried removing these ```:``` and it works.
So basically do you want `Args:` right?

Reply:
It is not that I want. It is the style that we had here before. If you want to remove the `:` you can try to follow https://numpydoc.readthedocs.io/en/latest/format.html more strictly then.